cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,123 +87,90 @@ describe("User Creation", () => {
-    userCreationPage.clearIntoElementById("weekly_working_hours");
-    userCreationPage.clickElementById("submit");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    userCreationPage.clickEditProfileButton();
+    userCreationPage.clearFirstName();
+    userCreationPage.clearLastName();
+    userCreationPage.clearPhoneNumber();
+    userCreationPage.clearAltPhoneNumber();
+    userCreationPage.clearWeeklyWorkingHours();
+    cy.submitButton(updateBtn);
     userCreationPage.verifyErrorMessages(EXPECTED_PROFILE_ERROR_MESSAGES);
   });
 
   it("create new user and verify reflection", () => {
-    userCreationPage.clickElementById("addUserButton");
-    userCreationPage.selectFacility("Dummy Shifting Center");
-    userCreationPage.typeIntoElementById("username", username);
-    userCreationPage.typeIntoElementById("password", "Test@123");
-    userCreationPage.selectHomeFacility("Dummy Shifting Center");
-    userCreationPage.typeIntoElementById("phone_number", phone_number);
-    userCreationPage.setInputDate("date_of_birth", "25081999");
-    userCreationPage.selectDropdownOption("user_type", "Doctor");
-    userCreationPage.typeIntoElementById("c_password", "Test@123");
-    userCreationPage.typeIntoElementById("qualification", "MBBS");
-    userCreationPage.typeIntoElementById("doctor_experience_commenced_on", "2");
-    userCreationPage.typeIntoElementById(
-      "doctor_medical_council_registration",
-      "123456789",
-    );
-    userCreationPage.typeIntoElementById("first_name", "cypress test");
-    userCreationPage.typeIntoElementById("last_name", "staff user");
-    userCreationPage.typeIntoElementById("email", "test@test.com");
-    userCreationPage.selectDropdownOption("gender", "Male");
-    userCreationPage.selectDropdownOption("state", "Kerala");
-    userCreationPage.selectDropdownOption("district", "Ernakulam");
-    userCreationPage.clickElementById("submit");
-    userCreationPage.verifyNotification("User added successfully");
+    userCreationPage.clickAddUserButton();
+    userCreationPage.selectFacility(homeFacility);
+    userCreationPage.typeUserName(username);
+    userCreationPage.typePassword(password);
+    userCreationPage.selectHomeFacility(homeFacility);
+    userCreationPage.typeNewUserPhoneNumber(phone_number);
+    userCreationPage.typeDateOfBirth(newUserDob);
+    userCreationPage.selectUserType(role);
+    userCreationPage.typeConfirmPassword(password);
+    userCreationPage.typeQualification(qualification);
+    userCreationPage.typeDoctorExperience(experince);
+    userCreationPage.typeDoctorMedicalCouncilRegNo(regNo);
+    userCreationPage.typeNewUserFirstName(newUserFirstName);
+    userCreationPage.typeNewUserLastName(newUserLastName);
+    userCreationPage.typeEmail(email);
+    userCreationPage.selectGender(gender);
+    userCreationPage.selectState(state);
+    userCreationPage.selectDistrict(district);
+    cy.submitButton(saveBtn);
```

Comment:
- on all form submission, make sure you intercept the API with cy.intercept() and verify they are giving 200

Reply:
Made the intercept for all form submission which includes the API call